### ISSUE TITLE
Unfortunate display of shared links on twitter

### ISSUE THREAD

**Issue by despens** [webrecorder/webrecorder#338]
![selection_999 320](https://cloud.githubusercontent.com/assets/571494/25283921/b6e06f52-26b5-11e7-941e-108b319544d6.png)

The link was https://webrecorder.io/despens/startup-prose/20170420200856/https://www.juicero.com/

So to a very specific page in the collection, but the summary contains info about the whole collection.

**Comment by m4rk3r** [comment]
@despens yeah, sometimes works well, other times not so much if the collection is broad. Right now if there isn't a collection description we default to the WR tagline.. maybe that should be the behavior regardless of description?

**Comment by despens** [comment]
Here is another example 
![selection_999 494](https://cloud.githubusercontent.com/assets/571494/26639179/670ec2ec-4624-11e7-8a54-4c3b6479937b.png)

Collection at https://webrecorder.io/despens/edballs/20150508121523/https://twitter.com/edballsmp/status/63623585020915713
